components/raftstore/src/coprocessor/mod.rs

```diff
@@ -232,9 +235,9 @@ impl CmdBatch {
 
 pub trait CmdObserver<E>: Coprocessor {
     /// Hook to call after preparing for applying write requests.
-    fn on_prepare_for_apply(&self, observe_id: ObserveID, region_id: u64);
+    fn on_prepare_for_apply(&self, cdc_id: ObserveID, rts_id: ObserveID, region_id: u64);
```

Comment:
```suggestion
    fn on_prepare_for_apply(&self, cdc_id: ObserveID, rts_id: ObserveID, region_id: u64);
```

Why do we need two ObserveID?

Reply:
One for cdc worker, one for resolved ts worker, commands will send to both these two workers respectively.